activemodel/test/models/book.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Book
+  include ActiveModel::Validations
+
+  attr_accessor :author, :title
+end
+
+class Author
+  include ActiveModel::Validations
+
+  attr_accessor :books, :name
+end
```

Comment:
I think we should move this to a seperate file like we do for the other models.
We only seem to include subclasses in the same file.